types/react-select/src/components/Control.d.ts

```diff
@@ -1,28 +1,27 @@
-import { ComponentType, ReactNode, Ref as ElementRef } from 'react';
+import { ComponentType, ReactNode, RefCallback, MouseEventHandler, TouchEventHandler } from 'react';
 
 import { borderRadius, colors, spacing } from '../theme';
 import { CommonProps, OptionTypeBase, PropsWithStyles } from '../types';
 
-interface State {
-  /** Whether the select is disabled. */
-  isDisabled: boolean;
-  /** Whether the select is focused. */
-  isFocused: boolean;
-}
-
-export type ControlProps<OptionType extends OptionTypeBase> = CommonProps<OptionType> &
-  PropsWithStyles &
-  State & {
-    /** Children to render. */
-    children: ReactNode,
-    innerRef: ElementRef<any>,
+export interface ControlProps<OptionType extends OptionTypeBase> extends CommonProps<OptionType> {
+    className?: string;
+    innerRef: RefCallback<HTMLDivElement>;
     /** The mouse down event and the innerRef to pass down to the controller element. */
     innerProps: {
-      onMouseDown: (event: React.MouseEvent<HTMLElement>) => void,
-    },
-  };
+      onMouseDown: MouseEventHandler<HTMLDivElement>;
+      onTouchEnd: TouchEventHandler<HTMLDivElement>;
+    };
+    /** Whether the select is disabled. */
+    isDisabled: boolean;
+    /** Whether the select is focused. */
+    isFocused: boolean;
+    /** Whether the select is expanded. */
+    menuIsOpen: boolean;
+    /** Children to render. */
+    children: ReactNode;
```

Comment:
Would you mind also adding the theme here in this PR? 
```suggestion
    children: ReactNode;
    /** The style theme */
    theme: Theme;
```

Reply:
The `theme` prop should go on the `CommonProps` interface not on the `ControlsProps` interface. I think it makes sense for that to be a separate PR.